README.md

```diff
@@ -145,9 +145,13 @@ things that authors can do to help keep the conversation moving along.
 
 The following labels may be applied to enhancements to help categorize them:
 
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/high`  indicates that the enhancement is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+newsletters.
+- `priority/medium`  indicates that the enhancement is related to a
+future release but it is not the next OCP release 
+- `priority/low` indicates that the enhancement is does not have a clear release date.
```

Comment:
I think priorit/low conflict with intention of closing PRs after a short period of time. perhaps same with medium.

Reply:
I think that makes sense

Reply:
Fixed it in https://github.com/openshift/enhancements/commit/fda887bedfd9cca56720a65d10fc07788be1b766. Take a look.